cli/src/main/scala/parser/CliParser.scala

```diff
@@ -69,11 +71,33 @@ class CliParser {
   }
   private def processGraph(config: CliParserConfig): Unit = {
     val source = Source.fromFile(config.input);
-    val deBruijnGraph = new DeBruijnGraph(LazyList(source.toString()), config.k);
+    val data = (for (line <- source.getLines()) yield line).toSeq
     source.close();
-    val writer = new PrintWriter(config.output);
-    writer.println("Here could be some info about graph");
-    writer.close();
+    if (config.output != null) System.setOut(new PrintStream(new File(config.output.getAbsolutePath)))
+    config.format match {
+      case "fasta" => {
+        val fasta = new Fasta();
+        val fastaData = fasta.read(data);
+
+        fastaData match {
+          case Left(string) => System.err.println(string)
+          case Right(record) => {
+            val deBruijnGraph = new DeBruijnGraph(LazyList(record.data), config.k);
+            System.out.println("geney-cli v. 0.1\n" + deBruijnGraph.path + "\n");
+          }
+        }
```

Comment:
Сборщик подключен к парсеру практически правильно, за исключением того, что для чтения файла надо использовать не `Fasta.read`, который читает лишь один рид, а `Fasta.reads`, который прочитает все риды. Плюс для записи последовательности в файл лучше использовать `Fasta.show` для правильного форматирования
```suggestion
        val fasta = new Fasta();
        val fastaData = fasta.reads(data);

        fastaData match {
          case Left(string) => System.err.println(string)
          case Right(records) => {
            val deBruijnGraph = new DeBruijnGraph(records.map(_.data).to(LazyList), config.k);
            System.out.println(fasta.show(Record(">geney-cli v. 0.1", deBruijnGraph.path)).mkString("\n"))
          }
        }
```
Аналогичные изменения нужны и для `Fastq`